modules/react.py

```diff
@@ -61,10 +61,11 @@ def sendmsg(source, recip, text):
         return "Vuoi pure che ti dica grazie? Gli altri ti odieranno per quello che hai fatto."
 
     text = text.lower()
-    for i in messages:
-        if text.rfind(i) != -1:
-            return messages[i]
-    return None
+    values = ()
+    for k in messages:
+        if text.rfind(k) != -1:
```

Comment:
Actually this is quite ugly, it can be replaced by ```if k in text:``` I guess I'll do it in the next commit if I don't forget.